backend/catalogue/admin.py

```diff
@@ -57,3 +63,34 @@ class RegionReportAdmin(admin.ModelAdmin):
         ('minx','maxx','miny','maxy',),
     )
 admin.site.register(models.RegionReport, RegionReportAdmin)
+
```

Comment:
Random minor one, but the general python spacing I try and stick with is "pep8", which is 2 blank lines between top-level classes/functions, and a single blank line in between eg class methods, or for readability.

Reply:
Understood, I will use pep8 as my guide from now om